apps/mobile/src/screens/welcome-screen/welcome-screen.tsx

```diff
@@ -1,11 +1,20 @@
 import React from "react";
 
-import { Text, View } from "~/libs/components/components";
+import { RadioGroup, Text, View } from "~/libs/components/components";
+
+const mockData = [
```

Comment:
Let's rename it to balanceWheelOptions as mockData looks to abstract here

Reply:
I think that's not an issue now. We will remove it anyway